docs/cli/update.md

```diff
@@ -0,0 +1,85 @@
+# Update
+This command used for updating DID.
+
+## Usage
+```
+update-did [flags]
+```
+
+## Flags
+* `domain` _[string]_ - URL to the TrustBloc consortium's domain.
+* `sidetree-url` _[array|string]_ - Array of 1 or more sidetree url.
+* `sidetree-write-token` _[string]_ - The sidetree write token.
+* `tls-cacerts ` _[array|string]_ - Array of 1 or more ca cert path.
+* `tls-systemcertpool ` _[boolean]_ - Flag whether to use system certificate pool.
+* `did-uri` _[string]_ - DID URI.
+* `add-publickey-file` _[string]_ - The File contains the DID public keys to be added or updated.
+* `remove-publickey-id` _[array|string]_ - Array of 1 or more public key id to be removed.
+* `add-service-file` _[string]_ - The File contains the DID services to be added or updated.
+* `remove-service-id` _[array|string]_ - Array of 1 or more service id to be removed.
```

Comment:
Array of one or more service IDs to be removed.

Reply:
done